### DISCUSSION TITLE
Use both regular and preemptible machines in Nextflow and Google Lifesciences GCP

### DISCUSSION THREAD

**Discussion by Abdelrahman-Galal** [nextflow-io/nextflow#2209] [Q&A]
Hi ,

I have a pipeline with multiple process that I am running on Google Cloud GCP using  Google Lifesciences executor.

I need some process to use preemptible machines while others not.

I tried to use "withName" and "withLabel" along with a configuration file but it did not work with me. 

Could you help to achieve that?

Thanks

**Comment by pditommaso** [comment]
Unfortunately, there isn't control at process level for the `preemptible` setting
